### DISCUSSION TITLE
change permissions of participants

### DISCUSSION THREAD

**Discussion by tsalzsieder** [te-online/timemanager#129] [Q&A]
Hey! 
I would like to change the rights different users have on eg one project. For example, I, as the admin, create the project and invite all relevant members to the team. The team lead, not me, should be able to see the reporting of the individual team members - is there any way to change the permissions for individual users? Can't find a way right now.

Any help is highly appreciated!

Best, Tom

**Comment by te-online** [comment]
Hey Tom 👋 

Your request makes total sense, however, the current version of the sharing feature doesn't support this use case 😢 

There have been quite a few requests for more fine-grained control, so it will probably land at some point, but it's not a priority right now.

Sorry that I can't help you with this right now.
